tests/spec/tests/init.spec.js

```diff
@@ -0,0 +1,79 @@
+const http = require('http');
+const cmd = require('../helpers/cmd');
+const rimraf = require('rimraf');
+const { existsSync, readFileSync } = require('fs');
+const yaml = require('js-yaml');
+
+describe('hs init', () => {
+  const { cli, config } = global;
+
+  it('should print the correct output for personal access key init', async () => {
+    rimraf.sync('hubspot.config.yml');
+    expect(existsSync('hubspot.config.yml')).toBe(false);
+    await cli.execute(
+      ['init'],
+      [cmd.ENTER, config.personalAccessKey, cmd.ENTER, 'QA', cmd.ENTER]
+    );
+
+    expect(existsSync('hubspot.config.yml')).toBe(true);
+
+    expect(
+      yaml.load(readFileSync('hubspot.config.yml', 'utf8')).portals[0]
+        .personalAccessKey
+    ).toEqual(config.personalAccessKey);
+  });
+
+  if (
```

Comment:
So we only conditionally run this test? Should we have some sort of console output to let the user know that it was skipped?

Reply:
I ended up breaking this file up into `init-personal-access-key.spec.js` and `init-oauth2.spec.js`. Also the tests assume that you have valid values for both auth flows.

I think although this is a bit more cumbersome, it will better to allow all tests to run.